tests/pfcwd/test_pfcwd_function.py

```diff
@@ -511,7 +511,7 @@ def __init__(self, ptf, router_mac, tx_mac, pfc_params, is_dualtor):
         if is_dualtor:
             self.vlan_mac = "00:aa:bb:cc:dd:ee"
         else:
-            self.vlan_mac = router_mac
+            self.vlan_mac = self.tx_mac
```

Comment:
let's add some commet.

For each topo t0/dual-tor/t1/t2: router_mac /tx_mac/vlan_mac what is expected value.